tests/time_all_tests

```diff
@@ -1,5 +0,0 @@
-#/bin/bash
-#time run_all_tests | grep time | sort -n -k 5 | gawk '{print; n++; sum+=$5} END { print "total time for",n,"tests = ",sum }'
-
-# Another option...
-time nosetests -s | grep time | sort -n -k 5 | gawk '{print; n++; sum+=$5} END { print "total time for",n,"tests = ",sum }'
```

Comment:
I use this occasionally to look for tests that are kind of slow.  Can we just switch it from `nosetests` to `pytest`.  I think the rest can be the same.

Reply:
Why not `pytest --durations=0` ?

Reply:
OK.  That works.  I didn't know about that feature.  I'll try to remember that next time I want to do this.